model/ddl.go

```diff
@@ -84,6 +84,7 @@ const (
 	ActionCreatePlacementPolicy         ActionType = 51
 	ActionAlterPlacementPolicy          ActionType = 52
 	ActionDropPlacementPolicy           ActionType = 53
+	ActionAlterTablePartitionOptions    ActionType = 54
```

Comment:
Maybe a smaller name. We will transform `ast.TableOption` into different actions, refer https://github.com/pingcap/tidb/blob/257140a154a13061b55993f499aa25f0cd07cdf6/ddl/ddl_api.go#L2530-L2573. We don't really have `AlterTableOption` type.

Reply:
> Maybe a smaller name. We will transform `ast.TableOption` into different actions, refer https://github.com/pingcap/tidb/blob/257140a154a13061b55993f499aa25f0cd07cdf6/ddl/ddl_api.go#L2530-L2573. We don't really have `AlterTableOption` type.

the constant is used for ddl.runDDLJob() to mark job type not ddl.AlterTable()